include/hipSYCL/sycl/libkernel/accessor.hpp

```diff
@@ -2250,6 +2267,11 @@ class local_accessor
       _num_elements{allocationSize}
   {}
 
+  specialized<address> getAddr() const { return _addr; }
```

Comment:
```suggestion
  specialized<address> AdaptiveCpp_get_addr() const { return _addr; }
```
Public functions that are not part of the spec are effectively AdaptiveCpp extensions. We usually prefix those with `AdaptiveCpp` :)